components/TravelMethodButtons/CarpoolCounter.jsx

```diff
@@ -1,86 +1,76 @@
-import {
-    Flex,
-    Text,
-    SimpleGrid,
-    GridItem,
-    Container,
-  } from "@chakra-ui/react";
-  
-  const Counter = ({ handleMinus, handlePlus, count }) => {
-    return (
-      <Container
-        width={["268px", "499px"]}
-        textAlign={["center", "left"]}
+import { Flex, Text, SimpleGrid, GridItem, Container } from "@chakra-ui/react";
+
+const Counter = ({ handleMinus, handlePlus, count }) => {
```

Comment:
please consider renaming this component to something more descriptive
```suggestion
const CarpoolCounter = ({ handleMinus, handlePlus, count }) => {
```